packages/create-redwood-app/template/redwood.toml

```diff
@@ -14,3 +14,5 @@
   port = 8911
 [browser]
   open = true
+[background]
+  updateChecks = true
```

Comment:
@jtoar @Tobbe Thoughts about naming? (And/or looping in Tom?) I think it should be something like:
- `notifications.newVersion`
- `versionUpgrades.latest`
- `redwoodVersionUpdate.latest`

?

Reply:
I imagined something like `cli.backgroundUpgradeChecks`, so yeah we need to loop Tom in I'd say.

I've been reviewing this PR yesterday and today with the aim of getting it in and I'm going to push back against adding toml support in this PR. We should have it eventually, but I wanted to merge this before 2023, and since we can't discuss it till then and adding a toml key let alone section is too big of a change to ship without discussing, let's just do it in a later PR.

So we should merge this PR where this feature is disabled by default, opt-in via the env var. Then come January, we'll add a toml key to CRWA to publicize it for real.

Reply:
> So we should merge this PR where this feature is disabled by default, opt-in via the env var. Then come January, we'll add a toml key to CRWA (enabled by default) to publicize it for real.

@jtoar I'm ok with this plan, but I don't understand the timeline. Additionally, instead of deleting the .toml related code from this PR and then starting from scratch in a new one, I'd recommend forking and creating a new PR so we can start from this state _and_ had a much better reminder about the next step to make this valuable.

Lastly, this:
`REDWOOD_BACKGROUND_UPDATE_CHECKS_ENABLED`

would be more clear as something like:
`NEW_VERSION_CHECK_ENABLED`

- "background" is confusing without context
- "redwood" is redundant

fwiw

Reply:
I named it `REDWOOD_ENABLE_UPGRADE_CHECKS`. Agreed that `BACKGROUND` isn't descriptive. But I think it has to start with `REDWOOD`, or at least indicate that it's for Redwood in somehow.

The timeline is:
- merge now, as soon as checks pass
- it will ship when it ships (in a minor)
  - we won't try to get people to use it yet, but it's there, and can be enabled if you want via `REDWOOD_ENABLE_UPGRADE_CHECKS`
- I put a topic on January's 3rd's meeting about this. We'll figure out the name of the toml section and variable then  
- We come back and update it January, with docs, create-redwood-app, etc.
- We work on the feature that shows upgrades/releases of a stable version if you're on an RC (this would change the logic that's there a bit)

Reply:
Just to mention that consistency of the env var naming is something I started to document at https://community.redwoodjs.com/t/consistent-env-variables/4143

Reply:
@thedavidprice new PR is done here: https://github.com/redwoodjs/redwood/pull/7162